2200027/4406.py

```diff
@@ -0,0 +1,8 @@
+T = int(input())
+
+alpha = ['a', 'e', 'i', 'o', 'u']
+for _ in range(1,T + 1):
+    text = input()
+    text = [i for i in text]

```

Comment:
```suggestion
    text = list(input())

```
위와 같이 작성할 수도 있을 거 같아요! 😄